platform/lib/common/filteredPage.js

```diff
@@ -122,9 +121,22 @@ class FilteredPage {
           }
         });
 
+    // Remove empty top level categories from sidebar
+    this._dom('.nav-item.level-1')
+        .each((index, navItem) => {
+          navItem = this._dom(navItem);
+
+          // ... consider a category empty if there are no links in it
+          if (!navItem.has('a').length) {
+            navItem.remove();
+          }
+        });
+
     // Remove eventually unnecessary tutorial dividers left by the
     // previous transformation
-    this._dom('.nav-item-tutorial-divider:last-child').remove();
+    /* eslint-disable max-len */
+    this._dom('.nav-item-tutorial-divider:last-child, .nav-item-tutorial-divider:first-child').remove();
```

Comment:
nit: I'd prefer splitting the string over disabling eslint.

Reply:
Definitely the better solution 😅